engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -1906,6 +1906,8 @@ namespace dmGameSystem
 
         dmRig::Result rig_res = dmRig::Update(world->m_RigContext, params.m_UpdateContext->m_DT);
 
+        WritePoseMatricesToTexture(world);
```

Comment:
I think this is a good fix.

For correctness, I think we should (later on) make sure we can verify that the async upload of the texture is actually done. Theoretically, we could still end up rendering and the texture hasn't finished uploading.

Reply:
But `WritePoseMatricesToTexture ` uses `SetTexture` not `SetTextureAsync`